site/layouts/shortcodes/orange-footer.html

```diff
@@ -22,15 +22,15 @@
 {{- $id := .Get "id" -}}
 {{- $input := .Inner -}}
 
-<footer class="footer bg-dark">
+<footer class="footer navbar" data-bs-theme="dark">
   <h2 class="visually-hidden">Sitemap & information</h2>
 {{- if $title_content }}
   <div class="container-xxl footer-title-content">
     <h3 class="footer-heading">Sign up to our mailing list</h3>
     <div class="row">
       <form class="d-flex col-12 col-md-9 col-lg-7 col-xl-6 col-xxl-5 gap-2 gap-md-3">
         <label for="inputEmail{{ $id }}" class="visually-hidden">Email</label>
-        <input type="email" class="form-control text-bg-dark border-dark" id="inputEmail{{ $id }}" placeholder="Enter your email">
+        <input type="email" class="form-control bg-transparent" id="inputEmail{{ $id }}" placeholder="Enter your email">
```

Comment:
We agreed that the inputs in footers and headers should have a `#141414` background color. There's probably a change to do here and in the headers. Missed that probably in the navbars review.